Sources/Sentry/include/SentrySDK+Private.h

```diff
@@ -19,6 +19,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 + (SentryHub *)currentHub;
 
++ (nullable SentryOptions *)options;
```

Comment:
`m`: What about changing this to the following so we can use the dot syntax?
```suggestion
@property (nonatomic, nullable, readonly, class) SentryOptions *options;
```